sample_app/source/tealium/tealiumBuilder.brs

```diff
@@ -16,7 +16,7 @@
 '@param logLevel as an integer 0-None,1-Errors,2-Warnings,3-Messages | higher log levels include lower
```

Comment:
Can we add periods to the end of these param descriptions?  The log level one actually looks like some text is cropped at the end.  Could add 'Default 3' to end as well. 

Is 'messages' clearer than 'verbose'?